services/aur/aur.service.js

```diff
@@ -168,8 +174,85 @@ class AurVersion extends BaseAurService {
   }
 }
 
+class AurMaintainer extends BaseAurService {
+  static get category() {
+    return 'social'
```

Comment:
The category for this one is a little tricky, but I'm thinking we should use a different one (maybe `other`). The social badges are a different breed and almost always have a very unique style (https://shields.io/category/social).

This is more of just an additional attribute badge for a package

Reply:
Good call! `others` is definitely a better fit.